Cartfile.private

```diff
@@ -1,2 +1,2 @@
-github "Quick/Quick" ~> 0.6.0
-github "Quick/Nimble" "v2.0.0-rc.3"
+github "Quick/Quick" ~> 0.9.2
+github "Quick/Nimble" "v4.0.1"
```

Comment:
Can we change this to using `~> 4.0` instead of pointing at a specific tag? That was only needed for the rc since Carthage ignores those.